Header header
float64 latitude
float64 longitude
float64 altitude
float64 utm_easting
float64 utm_northing
float64 fix_quality
float64 time
int8 zone
string letter
